### ISSUE TITLE
Request: publish release candidates on flathub-beta

### ISSUE THREAD

**Issue by krille-chan** [krille-chan/fluffychat#278]
#### Description

Currently FluffyChat's desktop release candidates are only published on Snap's `candidate` channel judging by https://gitlab.com/famedly/fluffychat/-/issues/767#linux-desktop-including-linux-phones and previous issues. I would like to install them from flathub-beta instead as I don't have Snap installed and I don't want to install it (due to everything being mounted and using space and permissions not handling external drives etc).

I didn't find the documentation on how to do this, but [Changes in Flathub land &#8211; Alexander Larsson](https://blogs.gnome.org/alexl/2019/02/19/changes-in-flathub-land/) has a *Publish beta releases!* section documenting it for end-users and pointing developers to [beta branch of flathub/org.godotengine.Godot at GitHub](https://github.com/flathub/org.godotengine.Godot/tree/beta) on how it's setup.

The linked post says

> This isn’t meant to be used for nightly builds, but for releases that has some level of testing and are expected to mostly work and be usable to non-developer end-users.

So I think it matches the Snap candidate channel and could be a good match for FluffyChat rc testing.

**Comment by anarcat** [comment]
heck, it would be nice to have the *stable* releases on flathub, as it stands it is missing a few releases past 0.17.0:

https://github.com/flathub/im.fluffychat.Fluffychat/pulls

**Comment by Mikaela** [comment]
I am still interested, while this too is likely more or less blocked by https://github.com/flutter/flutter/issues/41737.
